chef/data_bags/crowbar/migrate/network/010_add_rx_offloading.rb

```diff
@@ -0,0 +1,21 @@
+def upgrade ta, td, a, d
+  unless a.has_key? "enable_rx_offloading"
+    a["enable_rx_offloading"] = a["enable_tx_offloading"] || ta["enable_rx_offloading"]
+  end
+  unless a.has_key? "enable_tx_offloading"
+    a["enable_tx_offloading"] = ta["enable_tx_offloading"]
+  end
+  return a, d
+end
+
+
+def downgrade ta, td, a, d
+  unless ta.has_key? "enable_tx_offloading"
+    a.delete "enable_tx_offloading"
+  end
+  unless ta.has_key? "enable_rx_offloading"
+    a.delete "enable_rx_offloading"
+  end
+  return a, d
+end
+
```

Comment:
No need for the last empty line (or two lines between the two methods).

But this is nit-picky, so +1 and feel free to merge without fixing this :-)